apps-rendering/src/logger/clientProd/index.ts

```diff
@@ -1,16 +1,12 @@
 import type { LoggerFunctions } from '../LoggerFunctions';
 
 class ClientProdLogger implements LoggerFunctions {
-	// eslint-disable-next-line @typescript-eslint/no-empty-function -- To be implemented
```

Comment:
Why are we not longer linting against empty functions?

Reply:
i think this may be a stricter 'no unnecessary disabling comments' check

Reply:
Right, so you’re saying the ESLint feature to prevent empty functions was dropped a while back, and this is a remnant of when it used to be enforced? Makes sense… and one for the team to pick up again if we believe [`@typescript-eslint/no-empty-function` is valuable](https://typescript-eslint.io/rules/no-empty-function/)